src/components/entries/templating/FeelTemplating.js

```diff
@@ -0,0 +1,206 @@
+import { useCallback, useEffect, useState, useMemo, useRef } from 'preact/hooks';
+import { usePrevious, useStaticCallback, useShowEntryEvent } from '../../../hooks';
+import { isFunction } from 'min-dash';
+import { useError } from '../../../hooks';
+import classnames from 'classnames';
+import Description from '../Description';
+import FeelTemplatingEditor from './FeelTemplatingEditor';
+
+const noop = () => { };
+
+/**
+ * @param {Object} props
+ * @param {Object} props.element
+ * @param {String} props.id
+ * @param {String} props.description
+ * @param {Boolean} props.debounce
+ * @param {Boolean} props.disabled
+ * @param {String} props.label
+ * @param {Function} props.getValue
+ * @param {Function} props.setValue
+ * @param {Function} props.validate
+ */
+export default function FeelTemplatingEntry(props) {
+  const {
+    element,
+    id,
+    description,
+    debounce,
+    disabled,
+    label,
+    getValue,
+    setValue,
+    tooltipContainer,
```

Comment:
This is not added to the JSdocs. It would be interesting to see this live in action; I'd expect it's just a DOM element?

Reply:
Yeah so, I removed the JSdocs by accident from the codemirror editor. It's a DOM element or a DOM query that defines where tooltips are rendered, as we may want to render them at an entirely different layer than the editor.

Reply:
Should we add it to the JSdocs of `FeelTemplatingEntry` then? Just to be consistent.